src/explore-education-statistics-admin/src/components/ReleaseServiceStatus.tsx

```diff
@@ -118,6 +118,7 @@ const ReleaseServiceStatus = ({
       )}
 
       {currentStatus &&
+        Object.entries(currentStatus).length > 2 &&
```

Comment:
Why is the condition > 2?

Reply:
This is even weirder because the type for `currentStatus` is this:

```typescript
export interface ReleaseStatus {
  overallStage: string;
}
```